cmd/starter/c/starter.c

```diff
@@ -888,6 +888,10 @@ __attribute__((constructor)) static void init(void) {
 
     /* temporarily drop privileges while running as setuid */
     if ( sconfig->starter.isSuid ) {
+        /* value from the generated config.h file */
+        if ( SINGULARITY_SUID == 0 ) {
+            fatalf("You could not run setuid worfklow if Singularity was compiled with '--without-suid'\n");
```

Comment:
I'm going to guess the user is not going to be able to do anything with this message.

How does this error happen? Singularity was built using `--without-suid` and somehow `starter` now has the setuid bit set (because there should be no starter-suid in this case, right?)